### PR TITLE
feat(doge): CW-19851 add dogecoin

### PR DESCRIPTION
**Pull request by hankliu418**
----

*Checklist:*

- README.md includes:
  - [ ] The details of signing data and transaction data, this includes parameters.
  - [ ] Official docs or white paper.
  - [ ] Website or api can query assets and broadcast transaction.

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by cbx-tech** [comment]
Task linked: [CW-19851 Dogecoin SDK](https://app.clickup.com/t/25577573/CW-19851)

File: packages/coin-doge/README.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
+# CoolWallet Bitcoin SDK
+
+Typescript library with support for the integration of Bitcoin for third party application, include the functionalities of generation of addresses and signed transactions. 
+
+## Install
+
+```shell
+npm i @coolwallet/btc
```
Comment (tp26610) [inline packages/coin-doge/README.md:8]:
```suggestion
npm i @coolwallet/doge
```

File: packages/coin-doge/README.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+```
+
+## Usage
+
+```javascript
+import BTC from '@coolwallet/btc';
```
Comment (tp26610) [inline packages/coin-doge/README.md:14]:
```suggestion
import BTC from '@coolwallet/doge';
```

Comment (tp26610) [reply packages/coin-doge/README.md:14]: 下面有些 BTC 關鍵字，要在這個 PR 順便改成 Doge 嗎？

File: packages/coin-doge/package.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@coolwallet/btc",
```
Comment (tp26610) [inline packages/coin-doge/package.json:2]:
```suggestion
  "name": "@coolwallet/doge",
```

File: packages/coin-doge/README.md (hunk 2 above)
Comment (hankliu418) [reply packages/coin-doge/README.md]: 全部已調整完畢

**Review by tp26610 — APPROVED** [review]

**Review by gundomlegend — APPROVED** [review]

**Pull request merged**